functions/versioningV1V2GetNotification/src/app/eventHandler.js

```diff
@@ -115,11 +116,29 @@ exports.versioning = async (event, context) => {
     };
     return ret;
   } catch (error) {
-    const ret = {
-      statusCode: response?.status ?? 502,
-      body: response?.statusText ?? "problem calling fetch",
-    };
-    return ret;
+    if (error instanceof ValidationException) {
+      return {
+        statusCode: 400,
+        body: JSON.stringify(generateProblem(400, error.message))
+      }
+    } else {
+      console.error("Error on url " + url, error)
```

Comment:
@masciale89 console.warn invece che console.error (scatta l'allarme sull'api gateway quindi non serve in questo caso).